lib/mailserver.js

```diff
@@ -170,19 +170,25 @@ mailServer.create = function (
   mailDir,
   user,
   password,
-  hideExtensions
+  hideExtensions,
+  isSecure,
+  certFilePath,
+  keyFilePath
 ) {
   mailServer.mailDir = mailDir || defaultMailDir
   createMailDir()
 
   const hideExtensionOptions = getHideExtensionOptions(hideExtensions)
   const smtpServerConfig = Object.assign(
     {
+      secure: isSecure,
+      cert: certFilePath ? fs.readFileSync(certFilePath) : null,
+      key: keyFilePath ? fs.readFileSync(keyFilePath) : null,
       onAuth: smtpHelpers.createOnAuthCallback(user, password),
       onData: handleDataStream,
       logger: false,
       hideSTARTTLS: true,
-      disabledCommands: user && password ? ['STARTTLS'] : ['AUTH']
+      disabledCommands: user && password ? [] : ['AUTH']
```

Comment:
It looks like a default behavior has been modified here, and there may be some compatibility issues. 

Would it be better to handle a separate config configuration based on the secure status? Perhaps a simple discussion is needed here?

Reply:
Hey, thanks for checking this out. Good point. I think something like this would be more backwards compatible:

```javascript
disabledCommands: user && password ? (isSecure ? [] : ['STARTTLS']) : ['AUTH']
```

I can push this change if it looks okay.

Reply:
This looks good to me - I'll just add this change and get this merged!
 
```suggestion
      disabledCommands: user && password ? (isSecure ? [] : ['STARTTLS']) : ['AUTH']
```